Extensions/ParticleSystem/ParticleEmitterObject.h

```diff
@@ -184,6 +184,7 @@ class GD_EXTENSION_API ParticleEmitterBase {
     destroyWhenNoParticles = enable;
   };
 
+  float GetNbParticles() const { return nbParticles; };
```

Comment:
Not needed, this is the "editor" object data: you don't need to track how many particles there are... because you're not in the game ;)